.github/workflows/playwright.yml

```diff
@@ -13,6 +14,10 @@ jobs:
       USER_NAME: ${{ secrets.USER_NAME }}
       USER_EMAIL: ${{ secrets.USER_EMAIL }}
       USER_PASSWORD: ${{ secrets.USER_PASSWORD }}
+      CARD_NUMBER: ${{secrets.CARD_NUMBER }}
```

Comment:
I do not think it's a good idea to save a public card number. You can create an object in testData folder, and save these params there. But it is not an issue.